### ISSUE TITLE
Validating Multiple Forms Simultaneously on the Same Page (e.g., multi-step form, wizard, etc.)

### ISSUE THREAD

**Issue by stevewithington** [dlid/jquery-valjs#1]
I have multiple forms on one page, each within their own panel (e.g., multi-step form). However, only one form is visible at a time. When a user clicks save draft, next, or back, I want to validate all forms and highlight required form fields ... even if the form isn't visible at the moment. So, if they click on one of the other 'tabs' they'll see which fields are still required. I also have errors being appended to a bootstrap modal window that should appear, but at this point, I'm just hoping to get all forms validated simultaneously ... or within the same request.

I've tried a few different approaches, such as:

```
$('.btn-save-draft,.btn-back,.btn-next').on('click', function(e) {
    e.preventDefault();

    $('form.my-common-form-class').each(function() {
        var result = $(this).valjs('validateForm');
        console.warn(result);
    });
});
```

However, the result always comes back with no errors. Any suggestions on how else I should approach this?

**Comment by dlid** [comment]
Hi, sorry for the delay. Currently hidden elements are not validated at all. An option to validate hidden fields would be a very valid option in the validateForm method. I'll look into this!

**Comment by dlid** [comment]
Hi. I just added the option to include hidden fields when you use the validateForm method. This fix is included in version 1.2 and is available for download on the site (CDN version will be updated in a while).

http://valjs.io/docs/articles/methods/validateform

Please let me know if this does not solve your issue.
